### ISSUE TITLE
Add logging to the app

### ISSUE THREAD

**Issue by mimotej** [xvanick1/OqixTS#11]
It would be good for future debugging to add logging to the app.

**Acceptance criteria**
- [ ] Investigate logging methods
- [ ] Add one method to the app

**Comment by tkosci** [comment]
This could be covered by these two solutions.
Most used dedicated packages for logging are apparently [typescript-logging](https://www.npmjs.com/package/typescript-logging) and [tslog](https://tslog.js.org/#/). If we ignore javascript modules and plain console logging, the latter option would be propably better, since it seems easier to set up and more feature rich.

**Comment by mimotej** [comment]
> This could be covered by these two solutions. Most used dedicated packages for logging are apparently [typescript-logging](https://www.npmjs.com/package/typescript-logging) and [tslog](https://tslog.js.org/#/). If we ignore javascript modules and plain console logging, the latter option would be propably better, since it seems easier to set up and more feature rich.

I think best option is to use console logging no need for any additional dependencies for the project of this size.

**Comment by tkosci** [comment]
It would mean less dependencies, but wouldn't provide f.e. severity of the log entry and would potentialy create a tech debt with the project size increasing in the future.

**Comment by mimotej** [comment]
> It would mean less dependencies, but wouldn't provide f.e. severity of the log entry and would potentialy create a tech debt with the project size increasing in the future.

I don't think this project can get to the size where using simpler logging tools would create a tech debt (mind it contains other methods such as console.warn, console.error so it already has basic tools for logging). On the other hand it will ensure that if this project would get into the maintenance mode there would be one less library we need to keep updated...

**Comment by xvanick1** [comment]
@mimotej this issue still valid?

**Comment by mimotej** [comment]
Roselved via #130
